### ISSUE TITLE
How to use mongdb

### ISSUE THREAD

**Issue by cnkker** [skoruba/IdentityServer4.Admin#135]
How to use mongdb?help...

**Comment by RagingKore** [comment]
First and foremost, congrats on the project as it seems very interesting and a lot of hard work was put into it.

But actually @xmichaelx any document database is, in my opinion, at least 10x better than a relational one for this. Incredibly simple to use and maintain the total opposite of a relational database.

Anyway, I'm atm evaluating the project to see how simple it is too abstract all storage concerns so that any type of database can be used, by creating adapters, something that could have been done in the first place, but I guess EF was right there at hand.

I already have IdentityServer4 and AspNetCore Identity and Data Protection implementations for MongoDB, but how far the rabbit hole goes in id-svr admin I'm am yet to find out.

If doable, would this be of interest @skoruba @xmichaelx ?

**Comment by skoruba** [comment]
@RagingKore - For sure, if you can explore more details about mongo, feel free send some feedback or even PR. 😊
Thanks!

**Comment by skoruba** [comment]
Yeah, that is my task, merge all PRs, currently I am busy for reviews, but I will work on it ASAP. :)

Is it docker for you also important?

**Comment by sateeshs** [comment]
Any updates on Mongodb

**Comment by joesdu** [comment]
Any updates on Mongodb +1
